packages/components/psammead-most-read/src/Rank/index.jsx

```diff
@@ -37,53 +54,80 @@ const listHasDoubleDigits = ({ numberOfItems }) => numberOfItems >= 9;
 const columnIncludesDoubleDigits = (props, supportsGrid) =>
   isOnSecondColumn(props, supportsGrid) && listHasDoubleDigits(props);
```

Comment:
Should this function 
```js
const listHasDoubleDigits = ({ numberOfItems }) => numberOfItems >= 9;
```
Be more like
```js
const listHasDoubleDigits = numberOfItems => numberOfItems > 9;
```
Was thinking a few things:
1. Might not make sense to take an object where there's only one value used.
2. Would it not be greater than 9 rather than equal to or greater than 9?